Framework/Interaction/Tax.php

```diff
@@ -268,46 +268,6 @@ public function __construct(
         $this->customsConfig = $customsConfig;
     }
 
-    /**
-     * Return customer code according to the admin configured format
-     *
-     * @param \Magento\Quote\Api\Data\CartInterface|\Magento\Sales\Api\Data\OrderInterface $data
-     * @return string
-     */
-    protected function getCustomerCode($data)
```

Comment:
Can you explain why you got rid of this code, seemingly without replacing the logic that it implemented?

Reply:
See the "Data Mapping > Customer Code Format" config field here for the admin setting that controls this method: https://www.classyllama.com/wp-content/uploads/2017/03/configuration_screenshot_1.1.0.png

Reply:
Regard line https://github.com/classyllama/ClassyLlama_AvaTax/pull/180/files/dc9268607b8cce34aebe6f72007aa73e54e6af50#diff-defa2a9dc80d5ddb4732d944799e9648R397 and https://github.com/classyllama/ClassyLlama_AvaTax/pull/180/files/dc9268607b8cce34aebe6f72007aa73e54e6af50#diff-defa2a9dc80d5ddb4732d944799e9648R648